assets/package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "eslint --quiet \"src/**/*.ts*\"",
     "format": "eslint --fix --quiet \"src/**/*.ts*\"",
     "test": "jest",
-    "test:watch": "jest --watch -o",
```

Comment:
Found it really annoying we had a coverage report during test:watch that would scroll results off screen while working on tests, hope nobody minds me removing this here.